score-client/src/main/java/bio/overture/score/client/command/VersionCommand.java

```diff
@@ -17,8 +17,8 @@
  */
 package bio.overture.score.client.command;
 
+import static bio.overture.score.client.util.VersionUtils.getScmInfo;
```

Comment:
It would be nice to have all classes that we're moving from icgc.dcc into it's own score module. I see some of the new classes are duplicated on `score-client` and `score-server` maybe it's better to create a new module  `score-common` or so..